src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -465,11 +469,31 @@ If you pass data into an Appwrite Function, it'll be found in the request object
 This includes all invocation inputs from Appwrite SDKs, HTTP calls, Appwrite events, or browsers visiting the configured domain. 
 Explore the request object with the following function, which logs all request params to the Appwrite Console.
 
+### Request Types {% #request-types %}
+
+{% table %}
+* Request
+* Description
+---
+* `req.bodyRaw`
+* Returns the raw body text data.
+---
+* `req.bodyText`
+* Returns binary data that was converted into text.
```

Comment:
This is a little misleading because it says "returns binary data", but what does it return?